### ISSUE TITLE
Install required PLAnT workspace packages for UWG D3 demo

### ISSUE THREAD

**Issue by bsatoriu** [MAAP-Project/ZenHub#249]
Unmesh (username: 'greedo') is using 'wksp-plant'  for the D3 demo, which requires the following libraries:

conda install -c conda-forge cartopy
conda install -c conda-forge folium
conda install -c conda-forge pyproj
conda install -c conda-forge ffmpeg
conda install -c conda-forge gdal
conda install -c conda-forge pandas
conda install -c conda-forge scikit-learn
conda install -c conda-forge fiona
conda install -c conda-forge emacs
conda install -c anaconda scipy
conda install -c anaconda numpy

When loading his workspace, the following error is thrown, which prevents any MAAP plugins from loading in the workspace:

`AttributeError: module 'nbconvert.exporters' has no attribute 'WebPDFExporter'`

Since a fully-functional MAAP workspace is required for Unmesh's demo, we need to investigate which of these packages are causing runtime errors in the ADE PLAnT stack. I recommend first trying to install these packages in a separate PLAnT workspace and testing the workspace for errors.

Also see this note from Unmesh:

> The plant installation with conda is not the latest. Is there a way to install latest plant from
> git clone https://gitlab.com/plant/plant.git
> this (latest plant installation) is not needed right away. I can do that later too

**Comment by gchang** [comment]
WARNING! I see that the `jupyter_image` in repo.nasa points to the base images from mas.maap-project.org. That should be reconciled.

Also, given that we are moving to Che 7, I think we should just create an environment for him for the demo. That'll be less work since any new images built for him will be obsolete anyway.

**Comment by gchang** [comment]
Possible workaround is to install the packages immediately after the new workspace starts using the plant stable stack. Seems to work and I'm having Unmesh confirm. The key is to pin the jupyterlab version so that doesn't get upgraded with everything else.
```
cd /projects
conda install -c conda-forge jupyterlab=2.1.4 cartopy folium pyproj ffmpeg gdal pandas scikit-learn fiona emacs scipy numpy rasterio
# Upgrade plant
git clone https://gitlab.com/plant/plant.git 
cd plant
python setup.py install
# After all said and done, restart Kernel
```

**Comment by gchang** [comment]
The original problem with nbconvert is that the version he had in his workspace is 6.0.7. The stack is shipped with 5.6.1. His jupyterlab was also upgraded to 2.2.4 which probably screwed everything up.

**Comment by mayadebellis** [comment]
@gchang This is now fixed! `I see that the jupyter_image in repo.nasa points to the base images from mas.maap-project.org. That should be reconciled.`
